build.gradle

```diff
@@ -190,5 +198,8 @@ tasks.withType(Test).configureEach {
     useJUnitPlatform()
     testLogging {
         events "passed", "failed", "skipped", "standardError"
+        showStandardStreams = true
```

Comment:
Show the system.out and system.error in the test run log, easier investigation of issues in tests.